opencti-platform/opencti-front/lang/en.json

```diff
@@ -2550,5 +2550,8 @@
   "Settings default values": "Default value from entity settings: {value}",
   "Settings default values usage...": "If the data is not found in the CSV file then the default value set in Settings > Customization will be used instead. You can override this default value by filling the input above.",
   "No default value set in Settings...": "No default value set for this attribute in the entity settings. You can provide one with the input above. This value will be used if the data is missing in the CSV file.",
-  "Default values set": "Default values set"
+  "Default values set": "Default values set",
+  "Let the user choose marking definitions": "Let the user choose marking definitions",
+  "Use default marking definitions of the user": "Use default marking definitions of the user",
+  "Marking definitions to use by the csv mapper...": "Marking definitions to use by the csv mapper if some marking attributes have the policy set to: let the user choose the marking definitions."
```

Comment:
I found this text misleading, even more if you read too fast like I did
![image](https://github.com/OpenCTI-Platform/opencti/assets/3634942/84062ec3-e05a-4117-87e0-3c0bd96a0387)

I can try to propose another one:
```suggestion
  "Marking definitions to use by the csv mapper...": "Marking definitions used by the csv mapper only when some marking attributes have policy set to 'let the user choose the marking definitions.' If not this Marking definitions is ignored."
```
Is it possible to use bold for 'only ' or 'if'  whatever the texts is at the end ?

Maybe mine is too long :)